src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -109,58 +98,56 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
-          <dd className={styles.allTracks}>
-            <input
-              type="checkbox"
-              defaultChecked={applyToAll}
-              onChange={applyToAllToggle}
-            />
-            <label htmlFor="">All tracks</label>
-          </dd>
-        </dl>
-        <dl className="category">
-          <dd className="trackName">
-            <label htmlFor="">Track name</label>
-            <button className={styles.trackConfigSlider} onClick={toggleName}>
-              <img src={nameIcon} />
-            </button>
-          </dd>
-          <dd className="featureLabels">
-            <label htmlFor="">Feature labels</label>
-            <button className={styles.trackConfigSlider} onClick={toggleLabel}>
-              <img src={labelIcon} />
-            </button>
-          </dd>
-          <dd className={styles.heightSwitcher}>
-            <button className={styles.trackHeightBtn}>
-              <img src={trackHeightBtn} />
-            </button>
-          </dd>
-        </dl>
-        <dl className="category">
-          <dd className={styles.trackLock}>
-            <button className={styles.trackLockBtn}>
-              <img src={trackLockBtn} />
-            </button>
-          </dd>
-          <dd className="trackHighlight disabled">
-            <button className={styles.trackHighlightBtn}>
-              <img src={trackHighlightBtn} />
-            </button>
-          </dd>
-          <dd className="trackMove disabled">
-            <button className={styles.trackMoveBtn}>
-              <img src={trackMoveBtn} />
-            </button>
-          </dd>
-        </dl>
-      </section>
-    </div>
-  ) : null;
+  return (
+    <section className={styles.trackConfig}>
+      <dl className="category">
+        <dd className={styles.allTracks}>
+          <input
+            type="checkbox"
+            defaultChecked={applyToAll}
+            onChange={applyToAllToggle}
+          />
+          <label htmlFor="">All tracks</label>
+        </dd>
+      </dl>
+      <dl className="category">
+        <dd className="trackName">
+          <label htmlFor="">Track name</label>
+          <button className={styles.trackConfigSlider} onClick={toggleName}>
+            <img src={nameIcon} />
+          </button>
+        </dd>
+        <dd className="featureLabels">
+          <label htmlFor="">Feature labels</label>
+          <button className={styles.trackConfigSlider} onClick={toggleLabel}>
+            <img src={labelIcon} />
+          </button>
+        </dd>
+        <dd className={styles.heightSwitcher}>
+          <button className={styles.trackHeightBtn}>
+            <img src={trackHeightBtn} />
+          </button>
+        </dd>
+      </dl>
+      <dl className="category">
+        <dd className={styles.trackLock}>
+          <button className={styles.trackLockBtn}>
+            <img src={trackLockBtn} />
+          </button>
+        </dd>
+        <dd className="trackHighlight disabled">
```

Comment:
Can we use the imported `styles` from CSS module?

Reply:
Removed the classNames as they are not used anywhere.